Tests/CocoaPodsTest/PodInstall/Podfile.lock

```diff
@@ -52,19 +52,19 @@ EXTERNAL SOURCES:
     :path: "../../../"
 
 SPEC CHECKSUMS:
-  MapboxCommon: 4a0251dd470ee37e7fadda8e285c01921a5e1eb0
-  MapboxCoreMaps: eb07203bbb0b1509395db5ab89cd3ad6c2e3c04c
-  MapboxCoreNavigation: 6b3fffc8fe22a46896c2c3a2b6e1b021cf207842
+  MapboxCommon: 9de41fa77e21368ba7b9bfeee14bf71e80fd72f8
+  MapboxCoreMaps: 437928ba99b741ec4b92b6e48004881753feeb23
+  MapboxCoreNavigation: eaaa15a5b83c45f05bedff90625b98295ab7f0ce
   MapboxDirections: a620d78a54312c392323ba211df8ae7dd9a28ab4
-  MapboxMaps: af50ec61a7eb3b032c3f7962c6bd671d93d2a209
+  MapboxMaps: 0e1736688b9f4ad7b6b236616ffe66c4fcab0686
   MapboxMobileEvents: de50b3a4de180dd129c326e09cd12c8adaaa46d6
-  MapboxNavigation: bf21a38cd72ea182b0960ee63aab302d388d8405
-  MapboxNavigationNative: 904a6cc4a4bfe052b21fd1b8c529a9409a209098
+  MapboxNavigation: 6a35df35a755762046d0343e074d2766c4c7868d
+  MapboxNavigationNative: 1eaad981f94b52a24ad356490cce2aed9dc50c2a
   MapboxSpeech: cd25ef99c3a3d2e0da72620ff558276ea5991a77
   Polyline: 2a1f29f87f8d9b7de868940f4f76deb8c678a5b1
   Solar-dev: 4612dc9878b9fed2667d23b327f1d4e54e16e8d0
   Turf: 469ce2c3d22e5e8e4818d5a3b254699a5c89efa4
 
 PODFILE CHECKSUM: bde8103af0e9b326531ee57cf1fa935cbd5f2e18
 
-COCOAPODS: 1.12.1
+COCOAPODS: 1.12.0
```

Comment:
Downgrade?

Reply:
`Gemfile.lock` contained actually 1.12.0
I updated cocoapods version in Gemfile and reverted the change here.